gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/object.rb

```diff
@@ -305,7 +305,8 @@ def upload_file(source, options = {})
         uploader = FileUploader.new(
           multipart_threshold: uploading_options.delete(:multipart_threshold),
           client: client)
-        uploader.upload(source, uploading_options.merge(bucket: bucket_name, key: key))
+        response = uploader.upload(source, uploading_options.merge(bucket: bucket_name, key: key))
```

Comment:
Please add some documentation to this method about using the block feature. Also please limit this line to 80 characters like in [master](https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/object.rb#L317). You should be able to rebase against the latest changes.

(we plan to add back Rubocop which is why I'm starting to tackle violations slowly..)